docs/faq/perf.md

```diff
@@ -228,12 +228,12 @@ See [example/profiler](https://github.com/dmlc/mxnet/tree/master/example/profile
 for complete examples of how to use the profiler in code, but briefly, the Python code looks like:
 
 ```
-    mx.profiler.profiler_set_config(mode='all', filename='profile_output.json')
-    mx.profiler.profiler_set_state('run')
+    mx.profiler.set_config(profile_all=True, filename='profile_output.json')
+    mx.profiler.set_state('run')
```

Comment:
This API change shouldn't have happened. @cjolivier01 would you mind adding `profiler_set_state` back to fix the breaking change in #8972? cc'd @eric-haibin-lin who discovered the issue in #8972, @piiswrong who merged the breaking change.

Reply:
```https://github.com/apache/incubator-mxnet/tree/master/example/profiler``` also need to change.

Reply:
Changing the AP*I to the new code is desired